### PR TITLE
add python 3.9 tests

### PR DESCRIPTION
**Pull request by codeboten**
Python 3.9 wasn't being tested in this repo, fixing that.

### CODE REVIEW

**Review by lonewolf3739 — APPROVED** [review]
@codeboten docs ci is failing, your passionate area :)

File: tox.ini [hunk 1]
```diff
@@ -6,146 +6,146 @@ envlist =
     ; for specifying supported Python versions per package.
```
Comment (lzchen) [inline tox.ini:7]: Please rebase so you can add `instrumentation-base` :D

**Review by lzchen — APPROVED** [review]

File: tox.ini (hunk 1 above)
Comment (codeboten) [reply tox.ini]: done.

**Comment by codeboten** [comment]
> @codeboten docs ci is failing, your passionate area :)

@lonewolf3739 turns out the problem was that we pinned the version of Sphinx to an old version that contained a bug w/ python 3.9. fixed this by unpinning sphinx

**Pull request merged**